content/en/docs/release-oversight/improving-ci-signal.md

```diff
@@ -0,0 +1,125 @@
+
+##### Adverse Impacts
+Undetected (or detected but unresolved) regressions in payload health have a number of impacts:
+* The longer it takes us to detect it, the harder it is to determine which change introduced the issue because there are more changes to sift through
+* Unhealthy payloads can impact the merge rate of PRs across the org as they cause more test failures and generate distraction for engineers trying to determine if their PR has an actual problem
+* Consumers of bleeding edge payloads (QE, developers, partners, field teams) can’t get access to the latest code/features if we don’t have recently accepted payloads, so anything that causes more payload rejection to occur impacts our ability to test+verify other unrelated changes
+* Unhealthy CI data reduces confidence in test results.  Test failures become non-actionable when they become normalized.
+* Because all tests must pass in the same job run, we are very limited in which tests we can treat as gating (the tests we gate on must be highly reliable).
+
+### With Job Aggregation we now have...
+* Faster detection when we regress the health of our payload
+* Faster reaction to regressions (either delivering a fix to restore the health immediately, or reverting a change to provide more time to debug)
+* Ability to add more semi-reliable tests to our gating bucket and slowly ratchet up their pass rate while ensuring they do not regress further.  Our intention is for all nighly payload blocking jobs to be aggregated jobs.
+
+### Job Aggregation in a little more detail
+Instead of accepting payloads based on a “pass all tests once” gate, an aggregated job allows payloads to be accepted based on running all tests N times and ensures that, statistically, the observed pass percentage of each test is unlikely to have significantly regressed relative to our historical pass rate for each test, unless there is an actual regression present.
+
+Pay special attention to the distinction from previously focus on job success to focus comparing test pass rates within a job to their historical mean.  Another important thing to know is that, while these are primarily used for blocking nighly payload promotion, these aggregated jobs can optionally run (pre-merge).
+
+#### How are they used?
+Upon detection of a pass rate regression (As observed via attempted payload acceptance):
+* We find the PR that introduced the regression by opening revert PRs for all PRs that went in since the last good payload and running the payload acceptance check against each revert-PR.  The PR that passes the check will tell us which change was the source of the regression
+* While waiting for test results, a PR bot contacts all teams who own one of the PRs in question to make them aware their PR may be the source of a regression so they can begin investigating
```

Comment:
is this done via GH comment? who does it contact?

Reply:
Hmm... that bullet point is actually manual today.  This was pulled from the working doc and I failed to update that part.  Let me fix that.